RELEASE_NOTES.md

```diff
@@ -4,7 +4,18 @@ The release notes are typically a highlighting subset of all changes made. For f
 
 Note that version 2.1 is the first version with release notes. Please see the commit history for older versions.
 
-## NEXT_RELEASE
+## 3.0a2
+
+### Improvements
+
+- Minor improvements to build system, documentation and clean-up of obsolete code for FMI 3.0. In general this release contains no significant new user-related improvements compared to 3.0a1.
```

Comment:
We should mention that we fixed incorrect header in installation. That's quite significant since 3.01a was essentially broken.